i18nfield/strings.py

```diff
@@ -82,6 +82,12 @@ def localize(self, lng: str) -> str:
         else:
             return str(self.data)
 
+    def transform(self,f):
```

Comment:
Code-style-wise, I'd expect a space after the ``,``. Not sure if ``flake8`` will complain :)

Reply:
I didn't know this exists and is called this way in Python. I've renamed it :)